app/api/v2/models/orders_model.py

```diff
@@ -44,4 +44,27 @@ def get_all_orders(self):
 
         return None
 
-        self.db.close()
\ No newline at end of file
+        self.db.close()
+
+    def get_one_order(self, parcelId):
+        """"Method to fetch one parcel order delivery"""
+
+        self.db = db_init()
+        cur = self.db.cursor()
+        cur.execute("""SELECT user_id, user_id, pickup_location, destination, weight, price, status, date_created  FROM orders WHERE order_id = %s""", (parcelId, ))
+        all_orders= cur.fetchone()
```

Comment:
Good work @RachelleMaina, but I suggest you remove the print statement and ensure you close db connection before the return statement

Reply:
Will do this